governance-charter.md

```diff
@@ -138,9 +138,14 @@ around a more robust idea of community membership.
 ### Eligibility for candidacy
 
 Anyone may nominate either themselves or someone else to be a candidate in the
-election. To be ratified as a candidate, the nominee must accept the nomination
-and three Members of Standing (including the nominator, if she/he has standing)
-from three different employers, must endorse the nomination.
+election.
+
+To be ratified, candidates need three endorsements in total from members of
+standing from three different companies. When self-nominating, a candidate that
+is member of standing will need two more endorsements from members of standing
+from two other companies. When being nominated by a member of standing from a
+different company and accepting the nomination, the candidate that is a member
+of standing will have two endorsements already, needing only one more.
```

Comment:
I think we should be clear how such an endorsement looks like, in the past elections PRs from self-nominations got "thumbs ups", comments with "I support" or review approvals and it's unclear to me what counts. Since this is what we do already for people being added to the OpenTelemetry GitHub Org, I suggest that a comment like "I support" or "I endorse" should do it.

Reply:
+1 on making it clear. For the purpose of clarity and auditability, should that endorsement be a PR approval? Comments can be edited and not straight-forward to find their edit trail. PR approvals (even if the endorser is not an approver of the repo itself) cannot be modified. The comment in the approval can be changed, but not the approval itself after a PR is merged.

Reply:
We found PR approvals to be ambiguous, as the reviewer might be approving the markdown that is being added to the repository, not necessarily endorsing the candidate.

Reply:
I added a sentence here about being explicit by adding "I approve" as a comment.